doc/METADATA.ini

```diff
@@ -501,3 +501,16 @@ status= implemented
 usedby/plugins= csvstorage
 type= string
 description= used to manage the column order.
+
+[crypto/encrypted]
+status= implemented
+usedby/plugins= crypto
+type= string
+description= used to mark a key as encrypted. If value is not empty, the value has been encrypted.
+
+[crypto/encrypt]
+status= implemented
+usedby/plugins= crypto
+type= string
+description= used to mark a key for encryption. The crypto plugin is supposed to encrypt the key.
```

Comment:
can you document that it changes the semantics of "binary" (because it actually does not tell anymore if the key is actually binary)